app/src/main/java/fr/free/nrw/commons/media/MediaClient.java

```diff
@@ -38,13 +39,17 @@
 public class MediaClient {
 
     private final MediaInterface mediaInterface;
+    private final MediaDetailInterface mediaDetailInterface;
 
     //OkHttpJsonApiClient used JsonKvStore for this. I don't know why.
     private Map<String, Map<String, String>> continuationStore;
+    private static final String NO_CAPTION = "No caption";
```

Comment:
There is a string with that value in ContributionViewHolder.java already. Is it a coincidence? If I want to modify one, do I have to modify the other too?

Reply:
yes